src/mantarray_file_manager/files.py

```diff
@@ -30,11 +32,20 @@
 from .constants import UTC_FIRST_TISSUE_DATA_POINT_UUID
 from .constants import WELL_INDEX_UUID
 from .constants import WELL_NAME_UUID
+from .exceptions import FileAttributeNotFoundError
+from .exceptions import UnsupportedMantarrayFileVersionError
 from .exceptions import WellRecordingsNotFromSameSessionError
 
 PATH_OF_CURRENT_FILE = get_current_file_abs_directory()
 
 
+def _get_file_attr(h5_file: h5py.File, attr_name: str, file_version: str) -> Any:
```

Comment:
the `file_version` should be directly accessible from the `h5_file` object itself within this function, without needing to pass it explicitly right?